### PR TITLE
Project details page

### PR DESCRIPTION
**Pull request by juanbravozu**
## Changes

- Created project details page
- Fixed some spacing issues on categories
- Added more props to make ContentSection component more flexible

screencapture-localhost-3000-proyectos-detalle-reconociendo-tu-ciudad-2021-10-02-10_17_03

### CODE REVIEW

**Review by tatarodriguez11 — APPROVED** [review]

File: components/Hero.js [hunk 1]
```diff
@@ -25,7 +34,11 @@ export default function Hero({
       <div className="hero-shape" />
 
       <div className="hero-content">
-        <h1 className="hero-title">{sectionTitle}</h1>
+        <div className="hero-title-wrapper">
+          {iconUrl
+            && <img src={iconUrl} alt="" className="hero-icon" />}
```
Comment (Cruzlex7) [inline components/Hero.js:39]: shouldn't be that img tag in parenthesis to render?

File: pages/proyectos/detalle/[projectDetail].js [hunk 2]
```diff
@@ -1,28 +1,47 @@
...
     const { projectDetail } = context.query;
     const pageData = await getPageData(projectDetail, 'project');
+    const masterPageProps = await getPageData('', MASTERPAGE);
 
     return {
       props: {
         data: pageData,
         components: pageData.fields.projectDetail.fields,
+        masterPageProps: masterPageProps.fields,
+
```
Comment (Cruzlex7) [inline pages/proyectos/detalle/[projectDetail].js:19]: unnecessary new empty line

**Comment by juanbravozu** [comment]
> Now I'm sending from contentful a NavLink with the slug you need to match in the catgories set

Done :)

image

File: components/Navigation.js [hunk 3]
```diff
@@ -29,11 +33,21 @@ export default function Navigation({ fields }) {
         </button>
         <div className="navigation__list">
           <ul>
-            {navHeadre.map((link) => (
-              <li key={link.sys.id}>
-                <Link passHref href={`/${link.fields.link}`}>
-                  <a href="/" className={navigationType(link.fields.link)} onClick={handleCloseMenu}>
-                    {link.fields.label}
+            {navHeadre.map(({
```
Comment (tatarodriguez11) [inline components/Navigation.js:36]: no es navHeader en lugr de navHeadre?

**Review by tatarodriguez11 — APPROVED** [review]

**Pull request merged**